src/main/resources/config/jniconfig-java.json

```diff
@@ -62,13 +62,6 @@
       {"name":"longValue","parameterTypes":[] }
     ]
   }
-,
-  {
-    "name" : "java.lang.Object",
```

Comment:
We probably need to create an issue so that we can reintroduce this in case it is needed and in case https://github.com/oracle/graal/issues/3107 is fixed

Reply:
Filed https://github.com/gluonhq/substrate/issues/862